release/release.py

```diff
@@ -70,10 +70,12 @@
 import gpg
 import notes
 import preferences
-import sftp
```

Comment:
did we need to change from sftp to svn for this or is it just an additional change?

Reply:
we do not have any place to sftp to any more, now that `home.apache.org` is gone.